docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -966,13 +966,13 @@ allInsertsFuture.thenRun(() -> System.out.println("Seeded transaction events."))
 Create the anomalies tables:
 
 ```java
-String sql = "CREATE TABLE possible_anomalies WITH ("
+String sql = "CREATE SOURCE TABLE possible_anomalies WITH ("
```

Comment:
`CREATE SOURCE TABLE` does only work with a schema definition (aka `CT`), but not with a query (ie, not `CTAS`).

I think we can keep this example as-is, because the query computes more (eg, sum/count) than just doing a simple "upsert".